tableaudocumentapi/xfile.py

```diff
@@ -22,19 +22,23 @@ class TableauInvalidFileException(Exception):
 
 
 def xml_open(filename, expected_root=None):
+    """Opens the provided 'filename'. Handles detecting if the file is an archive,
+    detecting the document version, and validating the root tag."""
 
+    # Is the file a zip (twbx or tdsx)
```

Comment:
let's be consistent with what is in datasource.py and prepend each extension with a ".". so make this ".twbx or .tdsx" please